src/renault_api/cli/renault_vehicle.py

```diff
@@ -338,7 +369,7 @@ async def update_res_state(
 
 
 async def update_hvac_status(
-    vehicle: RenaultVehicle, status_table: Dict[str, str]
+    vehicle: RenaultVehicle, status_table: Dict[str, str], ctx_data: Dict[str, Any]
```

Comment:
The values of the `status_table` can now be anything. Please replace `str` with `Any` here:
```suggestion
    vehicle: RenaultVehicle, status_table: Dict[str, Any], ctx_data: Dict[str, Any]
```